Segue/Segue/SecondViewController.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  SecondViewController.swift
+//  Segue
+//
+//  Created by Limei  Chen on 10/12/21.
+//
+
+import UIKit
+
+class SecondViewController: UIViewController {
+
+    @IBOutlet weak var displayLabel: UILabel!
+    
+    @IBOutlet weak var tableview: UITableView!
+    var data: [String] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        if let dataToDisplay = data.first {
+            displayLabel.text = dataToDisplay
+        }
+        tableview.dataSource = self
+
+    }
+}
+
+extension SecondViewController: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 1
+    }
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = UITableViewCell()
```

Comment:
should always use tableView.dequeue and not create a cell with `UITableViewCell()`